omnibus/config/software/cleanup.rb

```diff
@@ -7,6 +7,10 @@
   # Delete cached .gem files and git checkouts
   delete "#{install_dir}/embedded/lib/ruby/gems/2.2.0/cache/*.gem"
   delete "#{install_dir}/embedded/service/gem/ruby/2.2.0/cache/*.gem"
+  delete "#{install_dir}/embedded/lib/ruby/gems/2.3.0/cache/*.gem"
+  delete "#{install_dir}/embedded/service/gem/ruby/2.3.0/cache/*.gem"
+  delete "#{install_dir}/embedded/lib/ruby/gems/2.4.0/cache/*.gem"
+  delete "#{install_dir}/embedded/service/gem/ruby/2.4.0/cache/*.gem"
```

Comment:
could this just be `.../ruby/2.*/cache/*.gem` so we don't have to update every ruby version bump?